docs/content/python/getting-started/quickstart/index.md

```diff
@@ -87,59 +85,29 @@ resource(_type: Page, "page", actions, roles) if
             permissions: ["read"]
         },
         admin: {
-            permissions: ["write"],
-            implies: ["guest"]
+            permissions: ["write", "read"]
         }
     };
  ```
 
-The `allow` rule is the top-level rule that we use to say who can do what in our application.
-In this case, we are delegating to Oso's built-in `role_allows` rule which implements all the
-authorization logic for role-based access control based on the data we provide in `actor_has_role_for_resource`
-and `resource`.
-
-An `actor_has_role_for_resource` rule looks up role objects that are associated with an actor.
-Role objects are of the form `{name: "the-role-name", resource: TheResourceObject}`.
-The Oso builtin roles will look up this rule, so this is required.
+The `allow` rule is what's queried by `is_allowed()` in your application.
+In this case, we are delegating to Oso's built-in `role_allows` rule.
+`role_allows` will use the data we provide in `actor_has_role_for_resource`.
 
-A `resource` rule governs access to a specific resource in your app — for instance, a page, an object, a route, or a database entry.
+A `resource` rule governs access to a specific resource in your app.
 In the above configuration for `Page`, there are two possible actions: `"read"` and `"write"`.
 Guests can read, but not write, to a `Page`.
 Admins are allowed to write to a `Page`.
 
-The line `implies: ["guest"]` says that admins can also do anything guests are allowed to do; that is, admins can also read a `Page`.
-
 There's much more you can do with Oso — we're just scratching the surface here. Polar is a very expressive language, and you can encode authorization logic far beyond what we've shown in this guide.
```

Comment:
Maybe we should make this more specific & help the reader see what else they could do with this type of policy.

You can define multiple `resource` rules, one for each resource type in your application that requires authorization. Resources can each have their own roles, or can inherit role & permission assignments from parent resources. To learn more see the roles getting started guide (maybe?).